ocp_resources/utils.py

```diff
@@ -276,3 +276,39 @@ def _return_resource(_resource, _check_exists, _msg):
                     _check_exists=check_exists,
                     _msg=skip_create_warn_msg,
                 )
+
+
+def add_rules(role_object):
+    """
+    Add a list of rules to a Role / CluterRole object.
+
+    Args:
+        role_object
+    """
+
+    def add_rule(
+        _role_object,
+        verbs,
+        api_groups,
+        permissions_to_resources,
+    ):
+        def _set_rule(rule_to_set):
+            _role_object.desired_state["rules"].append(rule_to_set)
+            _role_object.res["rules"] = _role_object.desired_state["rules"]
```

Comment:
any reason why this requires a separate function?

Reply:
I will move it to the loop in add_rules()

Reply:
I meant I'll move the content of it into the other function, so we will only have the add_rules().